qcodes/utils/slack.py

```diff
@@ -32,24 +42,24 @@
 performed every time `update()` is called.
 """
 
-import os
-import tempfile
-from functools import partial
-from time import sleep
 import inspect
-from slacker import Slacker
-import warnings
 import logging
+import os
+import tempfile
 import threading
 import traceback
+import warnings
+from functools import partial
+from time import sleep
+
 from requests.exceptions import ReadTimeout, HTTPError, ConnectTimeout
 from requests.packages.urllib3.exceptions import ReadTimeoutError
+from slack_sdk import WebClient
```

Comment:
shall we add this as an extra dependency to qcodes setup.cfg file? say, under (a new) "slack" section?

Reply:
Hi Mikhail, 

I added slack-sdk to the setup file. I specified the version that I used. There may be lower versions that work fine. How do you handle this?

Reply:
> There may be lower versions that work fine. How do you handle this?

i'd say, it's fine to use this version. the newer the versions - likely the better the library works. the only reason for specifiying an older version is to accomodate for inter-package requirements difficulties which may be easier to resolve if ëarlier" version is allowed. in other words, keeping the slack dependency "as is" is fine.